src/Components/Navbar/UserNavbar.js

```diff
@@ -121,12 +123,13 @@ export default function UserNavBar(props) {
                       Printing
                   </DropdownItem>
                   <DropdownMenu className='drp-menu'>
-                    <DropdownItem className='drp-item' href='/2DPrinting'>
-                        2D Printing
-                    </DropdownItem>
-                    <DropdownItem className='drp-item' href='/3DPrintingForm'>
-                        3D Printing
-                    </DropdownItem>
+                    {authedRoutes.map((link) => {
+                      return (
+                        <DropdownItem className='drp-item' href={link.route}>
```

Comment:
this will create the warning
```
Warning: Each child in a list should have a unique "key" prop
```
to fix:
```suggestion
                    {authedRoutes.map((link, index) => {
                      return (
                        <DropdownItem
                          key={index}
                          className='drp-item'
                          href={link.route}
                        >
```

to read more: [sentry.io](https://sentry.io/answers/unique-key-prop/)